mtgjson5/set_builder.py

```diff
@@ -433,6 +444,27 @@ def build_base_mtgjson_tokens(
     return build_base_mtgjson_cards(set_code, added_tokens, True)
 
 
+def add_sealed_uuid(mtgjson_set: MtgjsonSetObject) -> None:
+    """
+    Adds all uuids to each sealed product object within a set
+    :param mtgjson_set: the set to add sealed uuids to
+    """
+    for sealed_product in mtgjson_set.sealed_product:
+        add_uuid(sealed_product)
+
+
+def add_sealed_purchase_url(mtgjson_set: MtgjsonSetObject) -> None:
+    """
+    Adds all purhcase urls to each sealed product object within a set
+    :param mtgjson_set: the set to add purchase urls to
+    """
+    for sealed_product in mtgjson_set.sealed_product:
+        if sealed_product.identifiers.tcgplayer_product_id is not None:
```

Comment:
"is not None" isn't required, since we're just doing a simple empty / null check